### PR TITLE
MOTECH-2839: Added Cohort Query Member Task Trigger - Unit Tests

### PR DESCRIPTION
**Pull request by afijal**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: openmrs/src/test/java/org/motechproject/openmrs/tasks/builder/OpenMRSTriggerBuilderTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        when(configService.getConfigs()).thenReturn(configs);
+        triggerBuilder = new OpenMRSTriggerBuilder(configService);
+    }
+
+    @Test
+    public void shouldBuildProperTriggerEventRequests() {
+        List<TriggerEventRequest> triggerEventRequests = triggerBuilder.buildTriggers();
+        assertFalse(triggerEventRequests.isEmpty());
+
+        assertEquals(3, triggerEventRequests.get(0).getEventParameters().size());
```
Comment (kmadej) [inline openmrs/src/test/java/org/motechproject/openmrs/tasks/builder/OpenMRSTriggerBuilderTest.java:46]: you should verify if appropriate event parameters are added to trigger

**Comment by kmadej** [comment]
test this please

**Pull request merged**